src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/cas2/ApplicationNotesTransformer.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas2
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas2ApplicationNote
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.Cas2ApplicationNoteEntity
+
+@Component("Cas2ApplicationNoteTransformer")
+class ApplicationNotesTransformer {
+
+  fun transformJpaToApi(
+    jpa: Cas2ApplicationNoteEntity,
+  ):
+    Cas2ApplicationNote {
+    return Cas2ApplicationNote(
+      id = jpa.id,
+      name = jpa.createdByNomisUser.name,
+      username = jpa.createdByNomisUser.nomisUsername,
```

Comment:
Do we need to return the `nomisUsername` and `email` here? I'm not sure the frontend needs it for rendering the timeline, it just needs a name right? 

I also ask because that nomis username in particular is surely going to be a difference between nomis and external user objects. If we don't need to concern ourselves with it, I think we keep it simple and ignore it.

Reply:
External Users also have a `username` field, so they could be mapped to the same shape of data. But I think you're right that we're not using it on the frontend, so happy to take it out.